aws/resource_aws_alb_target_group_test.go

```diff
@@ -454,16 +454,18 @@ func TestAccAWSALBTargetGroup_updateLoadBalancingAlgorithmType(t *testing.T) {
 		CheckDestroy:  testAccCheckAWSALBTargetGroupDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccAWSALBTargetGroupConfig_loadBalancingAlgorithm(targetGroupName, false, ""),
+				// Set to non-default first
```

Comment:
Opted for some comments to try to emphasize the importance of test ordering here